nettests/ts-026-riseupvpn.md

```diff
@@ -58,29 +57,39 @@ If all parts of the API are functional and reachable then we write:
 ```json
 {
     "riseupvpn_api_failure": null,
-    "riseupvpn_api_status": "ok"
+    "riseupvpn_api_status": "ok",
+    "riseupvpn_ca_cert_status": true,
 }
 ```
 
 ## RiseupVPN gateways test
 
-When OpenVPN is installed, we can verify if a connection to these gateways is possible. If they can't be reached or we don't get the expected results we can presume them to be blocked. However, without OpenVPN installed, we don't run this test. We test by using the client certificate fetched in the API check.
+If the provider API is reachable, it provides a JSON-file which contains the IP addresses and capabilites of the VPN gateways. The reachability of gateways will be tested depending on their capabilities as described by the provider (ports, OpenVPN, obfs4) by performing TCP handshakes. If a TCP handshake fails we can presume the corresponding port and transport of that gateway to be blocked.
 
-If any one of the OpenVPN gateways are blocked then we presume them to blocked and write in the report: FIXME: DO WE WRITE WHICH GATEWAYS ARE BLOCKED? HOW?
+Example output for reported blocked gateways:
 
 ```json
 {
-    "riseupvpn_gateway_failure": "FAILURE STRING",
-    "riseupvpn_gateway_status": "blocked"
+   "riseupvpn_failing_gateways":[
```

Comment:
Shouldn't we use 192.0.2.0/24  as example IPs instead of real ones? It's nitpicking.

Reply:
ok